Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml

```diff
@@ -33,4 +33,4 @@ query: |
     | distinct Country, Actor 
   ) on Actor 
   | where set_Country !contains Country
-  | extend AccountCustomEntity = Actor, IPCustomEntity = IPaddress
+  //| extend AccountCustomEntity = Actor, IPCustomEntity = IPaddress
```

Comment:
same feedback on commenting

Reply:
Yeah, this is broken so commented it out while the author fixes it.